pkg/util/hostname_windows.go

```diff
@@ -0,0 +1,12 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package util
+
+import "os"
+
+func getSystemFQDN() (string, error) {
+	return os.Hostname()
```

Comment:
does this return a FQDN on windows?

Reply:
No it does not :/

Not sure what the best approach here is :
- Execute the os selection logic dynamically instead of at compile time
- Return an error for windows

It seems Agent5 wasn't trying to get a FQDN on windows : https://github.com/DataDog/dd-agent/blob/a7d1031b62301cd6e7c01cc50efbdf57b65c93ec/utils/hostname.py

WDYT ?

Reply:
either approach would work for me. I might have a slight preference for returning an error like `not implemented on Windows`, but up to you.